.vuepress/public/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Cosmos SDK Documentation",
```

Comment:
Cosmos SDK Tutorials @vince19972

Reply:
So sorry I forgot to change it again! Will always double check this part in the future.